src/pow.cpp

```diff
@@ -355,12 +355,16 @@ void multinomial_coefficients_mpz(int m, int n, map_vec_mpz &r)
 RCP<const Basic> pow_expand(const RCP<const Pow> &self)
 {
     RCP<const Basic> _base = expand(self->base_);
+    bool negative_pow = false;
     if (! is_a<Integer>(*self->exp_) || ! is_a<Add>(*_base))
         return self;
```

Comment:
You can see the stacktrace there, showing where the error is. 

Can you post code that you ran to trigger this? We should make sure it doesn't happen in master.